app/code/Magento/Widget/Block/Adminhtml/Widget/Options.php

```diff
@@ -236,6 +236,23 @@ protected function _addField($parameter)
         return $field;
     }
 
+    /**
+     * Get css classes from xml to apply for validation and/or css styles
+     *
+     * @param \Magento\Framework\DataObject $parameter
+     * @return string
+     */
+    protected function getCssClasses($parameter)
```

Comment:
Please, make method private and SVC build failures will be resolved

Reply:
I have made the changes suggested by you, but still it shows build failed. This time, it is showing that array_merge is being used in a forloop, which is right but actually it is not my change, default extension is having the same code. Can you please advice here what can be done ?